packages/grid/x-data-grid/src/colDef/gridDateOperators.ts

```diff
@@ -22,7 +22,7 @@ function buildApplyFilterFn(
 
   const time = new Date(year, month - 1, day, hour || 0, minute || 0).getTime();
 
-  return ({ value }): boolean => {
+  return ({ value }: { value: any }): boolean => {
```

Comment:
#4064 type this one correctly so we can keep this weak type for now

Reply:
@flaviendelangle Should I keep `any` here?

Reply:
Either are fine since it's going to change soon